popup/popups.js

```diff
@@ -0,0 +1,287 @@
+    badges: ['html', 'css', 'ruby on rails', 'javascript'],
+    liveBtn: 'See live',
+    sourceBtn: 'See Source',
+  },
+
+  {
+    id: 'popup4',
+    Title: 'Uber Navigation',
+    closeButt: '&times;',
+    Client: 'Uber',
+    counter1: 'images/Dot-shape.png',
+    Role: 'Lead Developer',
+    counter2: 'images/Dot-shape.png',
+    year: '2018',
+    snapshot: 'images/Snap4.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent'",
+    badges: ['html', 'css', 'ruby on rails', 'javascript'],
+    liveBtn: 'See live',
+    sourceBtn: 'See Source',
+  },
+];
+
+const modalArray = [];
+
+for (let i = 0; i < projects.length; i += 1) {
+  const mediaQuery = window.matchMedia('(max-width: 768px)');
+  if (mediaQuery.matches) {
+    const modal = document.createElement('div');
+    modal.classList.add('closePopup');
+    modal.classList.add('modalPty');
+
+    const card = document.createElement('div');
+    card.classList.add('cardPty');
+
+    modal.appendChild(card);
+
+    const title = document.createElement('h2');
+    title.textContent = projects[i].Title;
+    title.classList.add('titlePty');
+
+    const closeButt = document.createElement('span');
+    closeButt.innerHTML = projects[i].closeButt;
+    closeButt.classList.add('closeButtPty');
+
+    card.append(title, closeButt);
+
+    const counter1Img = document.createElement('img');
+    counter1Img.src = projects[i].counter1;
+    counter1Img.style.marginLeft = '5px';
+    counter1Img.style.marginRight = '5px';
+
+    const counter2Img = document.createElement('img');
+    counter2Img.src = projects[i].counter2;
+    counter2Img.style.marginLeft = '5px';
+    counter2Img.style.marginRight = '5px';
+
+    const positions = document.createElement('div');
+    positions.classList.add('positionsPty');
+
+    positions.append(projects[i].Client, counter1Img, projects[i].Role, counter2Img,
+      projects[i].year);
+
+    card.appendChild(positions);
+
+    const snapshot = document.createElement('img');
+    snapshot.classList.add('snapImages');
+    snapshot.src = projects[i].snapshot;
+    snapshot.style.width = '100%';
+    snapshot.style.height = '300px';
+
+    card.append(snapshot);
+    const description = document.createElement('p');
+    description.textContent = projects[i].description;
+
+    card.append(description);
+    description.style.width = '600px';
+
```

Comment:
- [ ] it would be better to stick to the design in order to complete the requirements here are improvements you need to fix 

- [ ] please make sure to use the right spacing like margin padding in order to have some space between elements like in the Figma design 

- [ ] please make sure to add font-weight, font-family font-size and the right color for the text 

- [ ] please make sure that the popup is responsive on  desktop version
- Yours 
<img width="1433" alt="Screenshot 2023-02-08 at 16 34 29" src="https://user-images.githubusercontent.com/61790587/217587060-bf353bc4-e216-44c8-8fbe-d3c32870a922.png">

- Figma design 
<img width="533" alt="Screenshot 2023-02-08 at 17 16 05" src="https://user-images.githubusercontent.com/61790587/217587197-a0f64ea3-7162-4e43-bd59-1b345414bbcb.png">